index.bs

```diff
@@ -1373,6 +1385,16 @@ To <dfn for="ReceiveStream">cancel</dfn> a {{ReceiveStream}} |stream|, run these
 1. Let |transport| be |stream|'s [=ReceiveStream/[[Transport]]=].
 1. Let |internalStream| be |stream|'s [=ReceiveStream/[[InternalStream]]=].
 1. Let |promise| be a new promise.
+1. Let |code| be 0.
+1. If |reason| is a {{WebTransportError}} and |reasons|'s [=[[ApplicationProtocolCode]]=] is not
```

Comment:
Need to add a |reason| argument to the algorithm.

Reply:
Thanks, done.